app/views/about/index.html.erb

```diff
@@ -141,4 +141,5 @@
     </div>
   </div>
 </div>
-</div>
+
+<%= slot('about-footer-slot', true) %>
```

Comment:
Needs to be inside `div#about` tag, so it does not show all the way down:

![](https://user-images.githubusercontent.com/2664090/54907879-e3096880-4ee6-11e9-8b73-cc08345865e0.png)